front/src/reducers/map/index.ts

```diff
@@ -177,21 +103,6 @@ export function updateViewport(
 export const mapSliceActions = mapSlice.actions;
 export type MapSliceActions = typeof mapSlice.actions;
 
-export const {
-  updateLayersSettings,
-  updateLineSearchCode,
-  updateMapSearchMarker,
-  updateMapStyle,
-  updateShowIGNBDORTHO,
-  updateShowIGNCadastre,
-  updateShowIGNSCAN25,
-  updateShowOSM,
-  updateShowOSM3dBuildings,
-  updateShowOSMtracksections,
-  updateTerrain3DExaggeration,
-  updateSmoothTravel,
-  updateViewportAction,
-  updateIssuesSettings,
-} = mapSliceActions;
+export const { updateMapSettings, updateViewportAction, updateIssuesSettings } = mapSliceActions;
```

Comment:
I think you don't need to export these actions:
- updateMapSettings should be retrieve by the hook
- updateViewportAction is not used
- updateIssuesSettings should come from the editor reducer and not the map reducer
```suggestion
```